shared/actions/wallets.js

```diff
@@ -95,19 +108,20 @@ const sendPayment = (state: TypedState) =>
 const requestPayment = (state: TypedState) =>
   RPCStellarTypes.localMakeRequestLocalRpcPromise(
     {
-      amount: state.wallets.buildingPayment.amount,
+      amount: state.wallets.building.amount,
```

Comment:
This doesn't support requests worth other currencies. I think it's as simple as checking if `building.currency` is set and passing in `currency` instead of `asset` if so.

Reply:
Hmm you mean if currency is set, we should pass an `Asset` of `building.amount`? It doesn seem to be in the store yet though

Reply:
The avdl for `makeRequestLocal` has 
```
union { null, Asset } asset, union { null, OutsideCurrencyCode } currency,
```
Right now we pass in `asset` but not `currency`. If `state.building.currency` is set we should send that in and switch `asset` to null